app/routes/model_utilities.py

```diff
@@ -0,0 +1,18 @@
+from flask import abort, make_response
```

Comment:
Thinking about the name of this file a bit more, what about `route_utilities` instead of `model_utilities`. While the functions we'll have in here do deal with models, the utility they provide is to the routes. `model_utilities` sounds like we might want to call these function _from_ models, which would be bad. These functions live at a different conceptual level from the models themselves. `validate_model` introduces flask dependencies (`abort` and `make_response`), which we would want to avoid doing in a model file.

Maybe the fact that the file lives in the routes folder is sufficient, but I'd still consider renaming it.

Reply:
I agree, it makes me feel a little weird to call it model_utilities since though it primarily creates models and filters them, it's also returning responses for routes and is generally not doing things that are _only_ model related.